### DISCUSSION TITLE
Init command

### DISCUSSION THREAD

**Discussion by route1rodent** [openupm/openupm-cli#350] [Ideas]
I am missing this init command to create the `Packages/manifest.json` for you, like npm does, so I can start a new project from a complete empty folder.

Thanks for this amazing tool

**Comment by favoyang** [comment]
Agreed!

**Comment by route1rodent** [comment]
what about also a `openupm init --dev` (or similar) for creating the `package.json` file of a new upm package? It could simplify the process, I remember it was not easy to understand how to fill that file properly, even with the Unity docs.

**Comment by favoyang** [comment]
Well, I would prefer `openupm init` for creating the package.json. And `openupm init --manifest` for the project manifest.

But I’m not sure what the manifest skeleton should contain, after all its already part of the process to create an Unity project.

**Comment by Thundernerd** [comment]
I am aware this is an old issue, however, Unity has a package in preview that does just this. They also create a default package.json so maybe that's something to take a look at?

The package in question can be found here: [https://docs.unity3d.com/Packages/com.unity.upm.develop@0.1/manual/index.html](https://docs.unity3d.com/Packages/com.unity.upm.develop@0.1/manual/index.html)

**Comment by favoyang** [comment]
@Thundernerd, thanks for the reference.

This issue is a low priority on my list. Ideally, I could wrap [yeoman](https://yeoman.io/) as `openupm init`. It is made for writing a generator and has a built-in way to handle interaction (query package name, display name...).
